commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/TwoSidedDrive.scala

```diff
@@ -8,6 +8,28 @@ import squants.{Acceleration, Dimensionless, Each, Length, Time, Velocity}
 import squants.motion.{MetersPerSecond, MetersPerSecondSquared}
 import squants.space.Meters
 
+trait TwoSidedDriveHardware extends UnicycleHardware {
+  val leftVelocity: PeriodicSignal[Velocity]
+  val rightVelocity: PeriodicSignal[Velocity]
+
+  val forwardVelocity: PeriodicSignal[Velocity] =
+    leftVelocity.zip(rightVelocity).map(t => (t._1 + t._2) / 2)
+}
+
+trait TwoSidedDriveProperties extends UnicycleProperties {
+  val maxLeftVelocity: Velocity
+  val maxRightVelocity: Velocity
+  val leftControlGains: VelocityGains
+  val rightControlGains: VelocityGains
+
+  val forwardControlGains: VelocityGains = PIDFConfig(
+    Each(0) / MetersPerSecond(1),
+    Each(0) / MetersPerSecondSquared(1),
+    Each(0) / Meters(1),
+    Each(0) / MetersPerSecond(0)
+  )
+}
+
 /**
   * A drivetrain with two side control (such as a tank drive)
```

Comment:
Actually, kinda. This is more of a package that includes the component, controllers, and tasks. That's why we have objects inside the trait, since we expect to mix in the traits into a package object so you get the feel of having a proper package with components, controllers, tasks, etc. inside.